### DISCUSSION TITLE
Why am I prompted to install a virus when running Interstellar independently?

### DISCUSSION THREAD

**Discussion by samer2011** [UseInterstellar/Interstellar#520] [Q&A] [answered]
<img width="467" alt="image" src="https://github.com/InterstellarNetwork/Interstellar/assets/160674984/b2f4adb9-19f9-4f7d-b665-8aa74600e604">

**Comment by bruhwhyamisobad** [comment, marked as answer]
> <img width="467" alt="image" src="https://github.com/InterstellarNetwork/Interstellar/assets/160674984/b2f4adb9-19f9-4f7d-b665-8aa74600e604">
> 
> 

These ads are just to pay for the servers that host Interstellar which costs over $105. Just click the "X" in the corner or close button (button is different on each ad) to close a ad. If you accidentally clicked an ad and it redirects to a website, just close the tab. You can also disable ads in settings.
